cypress/e2e/components/header.cy.js

```diff
@@ -49,27 +44,26 @@ describe('check headerimage and content', () => {
     });
     context('check topheader images', () => {
         it('check header image', () => {
-            cy.inHeader(() => {
-                const headerImage = cy.get('img.topHeader__image');
-                headerImage
+            cy.get(componentSelectors.topHeader).within(() => {
+                cy.get('img')
+
                     .should('be.visible')
                     .should('have.attr', 'src', 'img/header.jpg');
             });
         });
         it('check header logo', () => {
-            cy.inHeader(() => {
-                const logoArea = cy.get('div.topHeader__logoArea');
-                logoArea.should('be.visible');
-                logoArea.within(() => {
-                    cy.get('svg');
-                    cy.get('p').should('have.length', 2);
-                });
+            cy.get(componentSelectors.topHeader).within(() => {
+                cy.get(componentSelectors.logoArea)
+                    .should('be.visible')
+                    .within(() => {
+                        cy.get('svg');
```

Comment:
```suggestion
                        cy.get('svg').should('be.visible');
```
sonst testest du ja beim svg nichts

Reply:
akzeptiert und editiert